Hungarian/e107_plugins/news/news_months_menu.php

```diff
@@ -0,0 +1,126 @@
+<?php

```

Comment:
@yesszus what about this file? I think this is the wrong place... Please remove it. The rest is OK.